### ISSUE TITLE
Expand definition of "Plural"

### ISSUE THREAD

**Issue by kittensyzygy** [tatianamac/selfdefined#326]
Could the definition of "[Plural](https://www.selfdefined.app/definitions/plural/)" be expanded, perhaps with usage examples? As a user unfamiliar with the current definition in _Self-Defined_, I don't feel I understanding it after having read the definition. "[Boogaloo](https://www.selfdefined.app/definitions/boogaloo/)", to take another definition in comparison, gives the word's 'original meaning' and then expands the definition to include how it is used in other ways - might this be a useful type of thing to do for "Plural" too?

**Comment by kittensyzygy** [comment]
I've seen since that "Plural" is an "also read" in the definition "[Dissociative Identity Disorde](https://www.selfdefined.app/definitions/dissociative-identity-disorder/)". Perhaps the "Plural" definition could reference DID.

**Comment by tatianamac** [comment]
@kittensyzygy Definitely! All definitions can expand, particularly this one. For what it's worth, we want more definitions to be to the extent of Boogaloo, while recognising that definitions are a big and audacious task. So, sometimes we need shorter definitions to get started.

Good call on the 1:1 nature of the linking, though!
